packages/obonode/obojobo-chunks-list/viewer-component.scss

```diff
@@ -6,6 +6,18 @@
 		padding-left: 2em;
 		margin-bottom: 0;
 		margin-top: 0;
+
+		div {
+			margin: 0.5em 0;
+		}
+
+		div:first-child {
+			margin-top: 0;
+		}
+
+		div:last-child {
+			margin-bottom: 0;
+		}
```

Comment:
These changes don't seem like they're necessary in the scope of this issue. Is there a specific problem the extra margin is meant to solve?

If not, I recommend removing these rule changes.